src/templates/admin/submission/manager/delete_license.html

```diff
@@ -10,9 +10,12 @@
 {% block breadcrumbs %}
     {{ block.super }}
     <li><a href="{% url 'core_manager_index' %}">Manager</a></li>
-    <li><a href="{% url 'submission_licenses' %}">Licences</a></li>
     {% if license %}
```

Comment:
`license` will always be present on this template so we can remove the check here.

Reply:
logic for if license removed.